draft-ietf-quic-transport.md

```diff
@@ -5753,13 +5753,184 @@ DecodePacketNumber(largest_pn, truncated_pn, pn_nbits):
    return candidate_pn
 ~~~
 
+# Overview of Security Properties {#security-properties}
+
+A complete security analysis of QUIC is outside the scope of this document.  In
+this appendix, we provide an informal description of the desired security
+properties as an aid to implementors and to help guide protocol analysis.
+
+We cover properties of the handshake, general transport, and migration
+separately.
```

Comment:
I'd drop the use of "we."  In the whole `draft-ietf-quic-transport-22`, for instance, the word "we" is encountered only once, while RFC 7540 and 7541 contain no instances of this pronoun.

I suggest instead:
- _This appendix provides and informal description..._; and
- _Handshake, ...etc. are covered separately_.

Reply:
...and perhaps file an issue for that other use of "we."

Reply:
Removed all of the "we" from this diff, great suggestions as well, thanks!